web/src/components/tasks/business-formation/success/FormationSuccessPage.test.tsx

```diff
@@ -71,4 +71,10 @@ describe("Formation - <FormationSuccessPage />", () => {
     );
     expect(screen.queryByTestId(Config.formation.successPage.certifiedDocLabel)).not.toBeInTheDocument();
   });
+
+  it("shows the survey link", () => {
+    process.env.FEATURE_FORMATION_SURVEY = "true";
```

Comment:
dust - take a look at `OutageAlertBar.test.tsx` and add additional logic that temporary stores the env value, and replaces the process env back to the original state.

``````
  beforeEach(() => {
    jest.resetAllMocks();
    originalUrlValue = process.env.OUTAGE_ALERT_CONFIG_URL;
  });

  afterEach(() => {
    process.env.OUTAGE_ALERT_CONFIG_URL = originalUrlValue;
  });